app/helpers/user_helper.rb

```diff
@@ -50,6 +50,16 @@ def user_image_url(user, options = {})
     end
   end
 
+  def truncated_user_link(user, **options)
+    classes = %w[d-inline-block align-bottom text-truncate]
+    if options[:width]
+      style = "max-width: #{options[:width]}"
```

Comment:
I'm fairly sure that inline styles aren't allowed by our Content Security Policy.

Reply:
They aren't...